cmd/start-amazon-cloudwatch-agent/start-amazon-cloudwatch-agent.go

```diff
@@ -92,8 +93,25 @@ func main() {
 		log.Fatalf("E! Cannot translate JSON config into TOML, ERROR is %v \n", err)
 	}
 	log.Printf("I! Config has been translated into TOML %s \n", tomlConfigPath)
+	printFileContents(tomlConfigPath)
 
 	if err := startAgent(writer); err != nil {
+		log.Printf("E! Error when starting Agent, Error is %v \n", err)
 		os.Exit(1)
 	}
 }
+
+func printFileContents(path string) {
+	file, err := os.Open(path)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I think my original comment got lost. 

Do we want to `log.Fatal` here and cause a panic, or do we want to gracefully exit with `os.Exit()` like in the above function?

Reply:
Not my code but I think we should keep consistency here, I will change it to follow the existing pattern.